### ISSUE TITLE
TF400813 when triggering pipeline

### ISSUE THREAD

**Issue by dcp12345678** [huserben/TfsExtensions#246]
Getting the following error when trying to trigger a pipeline.  I have a pipeline called davetest1 and another called davetest2 which I am trying to trigger.  I have tried different authentication methods but I always get the TF400813 error.

We are using Azure Devops 2019 on prem.  

Using current Team Project
Team Project: PsfGlobalTeamProject with ID 2c3e8e4f-321e-4b85-971d-0ded11ad9f80
Using current Collection Url
Server URL: http://ourserver:8080/tfs/GitCollection/
Using following Authentication Method: Basic Authentication
Using Basic Authentication
##[error]﻿{"$id":"1","innerException":null,"message":"TF400813: Resource not available for anonymous access. Client authentication required.","typeName":"Microsoft.TeamFoundation.Framework.Server.UnauthorizedRequestException, Microsoft.TeamFoundation.Framework.Server","typeKey":"UnauthorizedRequestException","errorCode":0,"eventId":3000}
##[section]Finishing: Trigger a new build of davetest2

Here is the details of the task:
steps:
- task: benjhuser.tfs-extensions-build-tasks.trigger-build-task.TriggerBuild@4
  displayName: 'Trigger a new build of davetest2'
  inputs:
    buildDefinition: davetest2
    ignoreSslCertificateErrors: true
    authenticationMethod: 'Basic Authentication'
    username: 'my_username'
    password: mypassword
    checkbuildsoncurrentbranch: true

Thanks for any help.

**Comment by dcp12345678** [comment]
I was able to get this working for the basic authentication, but I had to enable basic authentication on the IIS side for the Azure DevOps application.  So if anyone else has this issue, try this solution.

Also, I was able to get the personal access token working as well.  It turns out I was putting the token name instead of the token value in the config for the task.  

Thanks for the great extension, it's really helpful.
